ses

```diff
@@ -65,9 +65,22 @@ if [ -x /usr/bin/ceph ]; then
     # remain, because their output is easier to read in a hurry ;)
     plugin_command "ceph --connect-timeout=$CT report" > $LOG/ceph/ceph-report 2>&1
     plugin_command "timeout $CT rados df" > $LOG/ceph/rados-df 2>&1
-    plugin_command "timeout $CT rbd ls" > $LOG/ceph/rbd-ls 2>&1
     plugin_command "radosgw-admin period get" > $LOG/ceph/radosgw-admin-period-get 2>&1
 
+    timeout $CT ceph osd pool ls detail 2>/dev/null |
+    sed -nEe "s/^.*'([^']+)'.* application rbd/\\1/p" |
+    while read pools; do
+        mkdir -p $LOG/ceph/images/$pools
+        timeout $CT rbd ls $pools 2>/dev/null \
+		> $LOG/ceph/images/rbd-ls-${pools} 2>&1 |
+	timeout $CT rbd -p $pools ls 2>/dev/null |
```

Comment:
nit: you could optimize this a little: instead of running `rbd ls` again just use the previous output: 
```
head -n ${OPTION_SES_RBD_INFO_MAX:=10} $LOG/ceph/images/rbd-ls-${pools} |
```

Reply:
If I do this, testing it fails saying the `$LOG/ceph/images/rbd-ls-${pools}` does not exist, unless I add a delay e.g. `sleep 2s` first, then it succeeds.

Reply:
I think it is because you have `|` above after the first `rbd ls $pools`, which is I suppose is just a typo, because I don't understand why it is there.